tests/transact_items_test.go

```diff
@@ -79,6 +79,46 @@ func TestTransactItems(t *testing.T) {
 				},
 			},
 		},
+		{
+			title:     "update multiple items with WithUpdateItem",
+			condition: "pk = :pk1 OR pk = :pk2",
```

Comment:
Dynamodb doesn't support `IN` or OR conditions for pk.
If you want to query multiple items, simply give them the same pk and different sk.